src/ssl.c

```diff
@@ -9048,6 +9087,17 @@ WOLFSSL_X509_EXTENSION* wolfSSL_X509_set_ext(WOLFSSL_X509* x509, int loc)
                 if (!isSet)
                     break;
 
+            #ifdef OPENSSL_ALL
+                ret = wolfSSL_ASN1_STRING_set(&ext->value, x509->subjAltNameSrc,
+                          x509->subjAltNameSz);
+                if (ret != WOLFSSL_SUCCESS) {
+                    WOLFSSL_MSG("ASN1_STRING_set() failed");
+                    wolfSSL_X509_EXTENSION_free(ext);
+                    FreeDecodedCert(&cert);
+                    return NULL;
+                }
+            #endif
+
```

Comment:
Can you also fix the leak cases below, since you are in this section of code and they have been identified? Lines 9105, 9119, 9129.